src/Common/RJSF/rjsfForm.scss

```diff
@@ -9,10 +18,45 @@
 
     &__array-field-item {
         grid-template-columns: 1fr auto;
+        z-index: 0;
+        background: transparent;
 
         & > :first-child {
             // This is added as the array field itself has a margin bottom
             margin: 0;
         }
+
+        &:has(> .remove-btn-container:hover)::after {
+            background: var(--R50);
+        }
+
+        &:has(> .remove-btn-container)::after {
+            border-radius: 4px;
+            transform: translate(-50%, -50%);
+            top: 50%; 
+            left: 50%; 
+            background: transparent;
+            z-index: -1;
+            position: absolute;
+            height: calc(100% + 12px);
+            width: calc(100% + 12px);
+            content: '';
+            transition: all 100ms ease-out;
+        }
+
+        & > .remove-btn-container:hover {
+            stroke: var(--R500);
+            path {
+                stroke: var(--R500);
+            }
+        }
+
+        & > .remove-btn-container {
+            stroke: var(--R500);
+            path {
+                stroke: var(--R500);
+            }
+            transition: all 100ms ease-out;
+        }
```

Comment:
I guess both of these can be combined into one

Reply:
yes I can remove the stroke and path from the hover class of remove-btn-container